src/js/components/Article.js

```diff
@@ -486,6 +501,11 @@ export default class Article extends Component {
       classes.push(this.props.className);
     }
 
+    let pagination;
+    if (this.props.pagination) {
```

Comment:
Are we showing pagination for Article with direction column? I believe the pagination is only applicable for `direction="row"`